test/beamchmark_test.exs

```diff
@@ -14,7 +14,7 @@ defmodule BeamchmarkTest do
   setup do
     temp_directory = Path.join([System.tmp_dir!(), "beamchmark_test"])
     on_exit(fn -> File.rm_rf!(temp_directory) end)
-    options = [delay: 0, duration: 1, output_dir: temp_directory, compare?: true]
+    options = [delay: 0, duration: 1, output_dir: temp_directory, compare?: true, formatters: []]
```

Comment:
I would add tests both for consol and html formatters.
Just to see if functions from Formatter behavior return properly.
In the case of HTML formatter we can also check output file size.